paasta_tools/setup_kubernetes_cr.py

```diff
@@ -298,12 +298,13 @@ def format_custom_resource(
                 paasta_prefixed("service"): service,
                 paasta_prefixed("instance"): instance,
                 paasta_prefixed("cluster"): cluster,
-                "paasta.yelp.com/eks": str(is_eks),
             },
             "annotations": {},
         },
         "spec": instance_config,
     }
+    if is_eks:
+        resource["metadata"]["labels"]["paasta.yelp.com/eks"] = str(is_eks)
```

Comment:
should this be
```suggestion
        resource["metadata"]["labels"][paasta_prefixed("eks")] = str(is_eks)
```
to be consistent with the other paasta.yelp.com labels?

Reply:
Thanks. updated the PR